tests/unit/test_arithmetic_expression.cpp

```diff
@@ -1169,6 +1169,15 @@ TEST_F(ArithmeticTest, CaseTest) {
 	result = AR_EXP_Evaluate(arExp, NULL);
 	AR_EXP_Free(arExp);
 	ASSERT_TRUE(SIValue_IsNull(result));
+
+	/* None of the alternatives evaluates to a none null value.
+	 * Default specified, expecting default. */
```

Comment:
Please expand on this, is `1` the default value ?
Further explain NULL handling in this case.